### PR TITLE
Llama-70b

### PR DESCRIPTION
**Pull request by mtairum**
### Ticket
#13368 

### Problem description
Part 3 of the Llama3 family of models bringup. 
(part 1: unified Llama3 codebase. Part 2: Llama3.2-11B multimodal)

### What's changed
- Add Llama3.1-70B to the unified Llama3 codebase which currently supports 1B/3B/8B/11B Llama3 models.
- `/models/demos/llama3/lt` tool is provided and updated to rapidly and interactively run any Llama3 tests and demos for any combination of weights and architectures 
- Update demo test to profile time to first token (prefill) and end to end decode performance, and push the data to the models dashboard. This works for the combination of all supported models and architectures.
- Fix compatibility issues with multimodal llama3.2.
 
### Checklist
- [x] [All post commit](https://github.com/tenstorrent/tt-metal/actions/runs/11681117516)
- [ ] [All T3k pipelines](https://github.com/tenstorrent/tt-metal/actions/runs/11669020806)
- [x] [All single chip](https://github.com/tenstorrent/tt-metal/actions/runs/11681113366/job/32525591641)
  - [x] [(previous all-single-chip)](https://github.com/tenstorrent/tt-metal/actions/runs/11597387885)
- [ ] [T3K perplexity](https://github.com/tenstorrent/tt-metal/actions/runs/11669023003)

### CODE REVIEW

**Review by mtairum — COMMENTED** [review]
-

File: models/demos/llama3/tests/multimodal/test_llama_image_attention.py [hunk 1]
```diff
@@ -76,9 +73,8 @@ def test_llama_attention_inference(batch, num_chunks, mesh_device, use_program_c
     mask = encoder_utils.build_encoder_attention_mask(pt_block_input, ar, ntok, num_chunks, 1)
     pt_block_input = pt_block_input.reshape(batch, -1, dim)
 
-    attention_input = prepare_inputs_ttnn_prefill(
+    attention_input = model_args.prepare_inputs_ttnn_prefill(
```
Comment (cglagovichTT) [inline models/demos/llama3/tests/multimodal/test_llama_image_attention.py:76]: probably want to force_replicated anywhere that you changed the vision tests

**Comment by mtairum** [comment]
We had most of the test passing last week (with a few exceptions that now have been fixed).

Given this I've re-run all the pipelines. Will merge after they show all green.

File: .github/workflows/t3000-demo-tests-impl.yaml [hunk 2]
```diff
@@ -10,9 +10,9 @@ jobs:
       matrix:
         test-group: [
           { name: "t3k_falcon40b_tests", arch: wormhole_b0, cmd: run_t3000_falcon40b_tests, timeout: 50, owner_id: U053W15B6JF}, #Djordje Ivanovic
-          { name: "t3k_llama3_70b_tests", arch: wormhole_b0, cmd: run_t3000_llama3_70b_tests, timeout: 30, owner_id: U03FJB5TM5Y}, #Colman Glagovich
```
Comment (tt-rkim) [inline .github/workflows/t3000-demo-tests-impl.yaml:13]: Haha not sure why this was moved around but whatever

**Review by tt-rkim — APPROVED** [review]

**Review by cglagovichTT — APPROVED** [review]

**Review by johanna-rock-tt — APPROVED** [review]

**Review by yieldthought — APPROVED** [review]

File: .github/workflows/t3000-demo-tests-impl.yaml (hunk 2 above)
Comment (mtairum) [reply .github/workflows/t3000-demo-tests-impl.yaml:13]: Just to fix the order of smaller llama3 to large llama3 :) 
I know it doesn't change the order of pickup by a machine, but makes more sense when looking at the tests in the pipeline.

**Review by ttmchiou — APPROVED** [review]

**Pull request merged**